react/index.tsx

```diff
@@ -4,15 +4,23 @@ import { ProductOrder, Impression, PixelMessage } from './typings/events'
 export function handleEvents(e: PixelMessage) {
   switch (e.data.eventName) {
     case 'vtex:userData': {
-      const { user } = e.data
-      epica('identify', user.id, {
-        isAuthenticated: user.isAuthenticated,
-        firstName: user.firstName,
-        lastName: user.lastName,
-        document: user.document,
-        id: user.id,
-        email: user.email,
-        phone: user.phone,        
+      const {
+        isAuthenticated,
+        firstName,
+        lastName,
+        document,
+        id,
+        email,
+        phone,
+      } = e.data
+      epica('identify', id, {
```

Comment:
This property might be `undefined` if the user is not logged in.

I think we should only trigger this event if the user is logged in. What do you think?

Reply:
I agree with you, thanks for the observation.